bcda/cli.go

```diff
@@ -324,6 +327,7 @@ func importCCLFDirectory(filePath string) (success, failure, skipped int, err er
 			continue
 		} else {
 			log.Infof("Successfully imported CCLF0 file: %v", cclf0)
+			cclf0.imported = true
```

Comment:
What happens if the CCLF0 file imported successfully, but the 8 and 9 were not yet present on our system (they hadn't been sent yet)?  If that happens, then because CCLF0 has `imported=true`, it will be cleaned up.  The 8 and 9 will never be retried properly on the next ingestion, because the CCLF0 was already removed.  I think they should be marked as a group, instead of individually.